src/v/redpanda/application.cc

```diff
@@ -814,7 +815,8 @@ void application::wire_up_redpanda_services() {
       std::ref(partition_manager),
       std::ref(controller->get_topics_state()),
       &kafka::make_consumer_offsets_serializer,
-      std::ref(config::shard_local_cfg()))
+      std::ref(config::shard_local_cfg()),
+      kafka::enable_group_metrics::yes)
```

Comment:
clarification: should this be `no`?

Reply:
exactly `_co_group_manager` is the new one, so metrics are enabled.